### PR TITLE
Remove yaml js dependency

### PR DESCRIPTION
**Pull request by TheTincho**
This removes the need to parse YAML in JavaScript, by providing a pre-parsed configuration in the JSON data returned in `/api/v1/status`.

### CODE REVIEW

**Comment by TheTincho** [comment]
Adding @fabxc in case it is needed. (sorry for the noise if it is not)

File: api.go [hunk 1]
```diff
@@ -131,10 +139,12 @@ func (api *API) status(w http.ResponseWriter, req *http.Request) {
 
 	var status = struct {
 		Config      string            `json:"config"`
+		ConfigJson  config.Config     `json:"configJson"`
 		VersionInfo map[string]string `json:"versionInfo"`
 		Uptime      time.Time         `json:"uptime"`
 	}{
 		Config: api.config,
+		ConfigJson: api.configJson,
```
Comment (fabxc) [inline api.go]: Capitalized `JSON` for both.

Comment (TheTincho) [inline api.go]: Done. I also changed the visible JSON object member, please tell me if you prefer to keep the old one.

**Comment by fabxc** [comment]
Okay, can you squash the commits and run `make assets` and `make format` before?
Good for merge then.

👍

**Comment by TheTincho** [comment]
All is done now, but the CI seems to have some issues. I presume some test is being flaky.

**Pull request merged**